android/app/build.gradle

```diff
@@ -350,12 +350,6 @@ android {
         exclude 'META-INF/LICENSE.txt'
         exclude 'META-INF/NOTICE.txt'
     }
-
-    // Use Java 1.8
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
```

Comment:
@gmaclennan do you remember the context for having this? I see that it was introduced in https://github.com/digidem/mapeo-mobile/commit/68a3136835836895190ad344d721a7f9b4d221bd but it causes the detox build command to break, and removing it entirely seems to have no other consequences.

Reply:
This was the "recommendation" with the setup of expo-unimodules for the version we are currently using (v0.3.1): https://github.com/unimodules/react-native-unimodules/blob/v0.3.1/README.md?plain=1#L62

It is unclear what impact not having this will have, since the statement is just "we recommend", so if this works, then I think we are ok. You should check the camera works ok in the Release Candidate QA I think.